### DISCUSSION TITLE
node and connector option: shows on top

### DISCUSSION THREAD

**Discussion by petersun94** [freeplane/freeplane#266] [Help] [answered]
last night, I was handling two nodes with their detail text, from which I wanted to connect keywords of both  for marking important information.

It occurred to me that I can create a tiny node with background color then place it above details area like this:

![image.png](https://pic.rmb.bdstatic.com/bjh/f09dea1b600b19294083a409e1cb273a.jpeg)

It seems great, but later I found a problem:
tiny node and connector would be covered by other node / background color / image

1. background color

tiny node and connector would be covered by background color:
connector was covered:

![image.png](https://pic.rmb.bdstatic.com/bjh/3fe151dfadb95e47ad8272f825ad8637.jpeg)

and select main node, the tiny node was covered too.

![img](https://pic.rmb.bdstatic.com/bjh/7a8198ebcea9cd0f78c9a9c348e5a72e.jpeg)

2. other situation:

before select them:

![image.png](https://pic.rmb.bdstatic.com/bjh/7f855c9c14583830d2406464f65c4d98.jpeg)

after select them, tiny node was covered:

![image.png](https://pic.rmb.bdstatic.com/bjh/8494badac69dd07e949eee749f93b151.jpeg)

so, my question is. what do you think of this idea, and is it possible to add a option to let node and connector shows on top?
thanks.

**Comment by EdoFro** [comment, marked as answer]
there is an option in preferences:

`Preferences…->Defaults->Connectors->Paint connectors behind nodes
`

uncheck it and see if it works for your use case

cheers,
edo

**Reply by petersun94** [reply]
thank you very much. I am sorry for missing it, and I also glad there exists solution already.
